src/main/java/Echoa/Event.java

```diff
@@ -0,0 +1,87 @@
+package Echoa;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+/**
+ * Event is a class that encapsulates the characteristics of an Event Task.
+ * It extends from the class Task,
+ * and contains additional characteristics of
+ * startTime and endTime
+ */
+
+public class Event extends Task {
+
+    private LocalDateTime startDateAndTime;
+    private LocalDateTime endDateAndTime;
+
+    public Event(String description, LocalDateTime startDateAndTime, LocalDateTime endDateAndTime) {
+        super(description);
+        this.startDateAndTime = startDateAndTime;
+        this.endDateAndTime = endDateAndTime;
+    }
+
+    public Event(String description, boolean isDone, LocalDateTime startDateAndTime, LocalDateTime endDateAndTime) {
+        super(description, isDone);
+        this.startDateAndTime = startDateAndTime;
+        this.endDateAndTime = endDateAndTime;
+    }
+
+    public LocalDateTime getStartDateAndTime() {
+        return this.startDateAndTime;
+    }
+
+    public LocalDate getStartDate() {
+        return this.getStartDateAndTime().toLocalDate();
+    }
+
+    public LocalTime getStartTime() {
+        return this.getStartDateAndTime().toLocalTime();
+    }
+
+    public LocalDateTime getEndDateAndTime() {
+        return this.endDateAndTime;
+    }
+
+    public LocalDate getEndDate() {
+        return this.getEndDateAndTime().toLocalDate();
+    }
+
+    public LocalTime getEndTime() {
+        return this.getEndDateAndTime().toLocalTime();
+    }
+
+    /**
+     * The method reformat the given date into MONTH-DD-YYYY.
+     *
+     * @param date LocalDate to be reformatted.
+     * @return String representation of reformatted date.
+     */
+    public static String getReformattedDate(LocalDate date) {
+        return date.getMonth() + " " +
+               date.getDayOfMonth() + " " + date.getYear();
+    }
```

Comment:
I noticed the same method on the Deadline class as well. Would it be better if it was placed on the Task class since its a static method?